crates/symbolicli/src/main.rs

```diff
@@ -0,0 +1,180 @@
+use std::io::Write;
+use std::{path::PathBuf, sync::Arc};
+
+use symbolicator_service::config::Config;
+use symbolicator_service::types::Scope;
+use symbolicator_sources::{SentrySourceConfig, SourceConfig, SourceId};
+
+use anyhow::{bail, Context};
+use clap::Parser;
+use reqwest::{header, Url};
+use serde::Deserialize;
+use tempfile::{NamedTempFile, TempPath};
+
+const BASE_URL: &str = "https://sentry.io/api/0/";
```

Comment:
we should also make this configurable for self hosted or single tenant usage.

Reply:
Yes, definitely.